src/AuthError.ts

```diff
@@ -1,17 +1,46 @@
-export class AuthError extends Error {
-  constructor(message: string, public code: AuthErrorCodes, public originError?: unknown | Error) {
-    super(message);
-  }
+import { ConnectJwt } from './types';
+
+export interface InvalidQshInfo {
+  computed: string;
+  received: string;
 }
 
-export enum AuthErrorCodes {
-  MISSED_TOKEN = 'MISSED_TOKEN',
+export enum AuthErrorCode {
+  MISSING_JWT = 'MISSING_JWT',
```

Comment:
I have fixed some codes with what I think it's more accurate English.

Reply:
This is definitely an issue I faced myself after renaming this.
Once this PR is good I will compile the list of breaking changes and perhaps even a migration guide if that makes sense.